BackEnd/src/business/sheets/draftParser.ts

```diff
@@ -1,11 +1,30 @@
 import { RisenSheetParser, RisenTeam } from './sheets';
-// TODO when we have reqs for draft
+import { getPlayerIdentifierFromString, hasValidOPGG } from './parserHelpers';
 export class DraftParser implements RisenSheetParser {
   public buildTeam(row: any[]): RisenTeam {
-    return null;
+    return {
+      teamName: row[0],
+      abrv: row[1],
+      win: row[2],
+      loss: row[3],
+      top: getPlayerIdentifierFromString(row[7]),
+      jungle: getPlayerIdentifierFromString(row[9]),
+      mid: getPlayerIdentifierFromString(row[11]),
+      adc: getPlayerIdentifierFromString(row[13]),
+      support: getPlayerIdentifierFromString(row[15]),
+      sub1: undefined, // No defined subs in draft league
+      sub2: undefined,
+      sub3: undefined,
+      sub4: undefined,
+      sub5: undefined,
+    };
   }
 
   public isValidRow(data: string[]): boolean {
-    return false;
+    return data.length > 5 && this.isValidRowData(data[0]) && this.isValidRowData(data[1]) && hasValidOPGG(data[17]);
```

Comment:
I'm reading opgg's on 15 https://docs.google.com/spreadsheets/d/1Af6MlaxpZkKWokvsi1cmBNl1L0tkP2Ld8aow9s45U2g/edit?gid=736224548#gid=736224548

Reply:
Thats a premade sheet, this is a draft league parser